### DISCUSSION TITLE
No way to navigate to links in Markdown Visual notes

### DISCUSSION THREAD

**Discussion by maiertech** [standardnotes/forum#2582] [Suggestions]
**Describe the bug**

When I pull up a Markdown Visual note that contains links, there is no way to just click the link. I need to copy paste the link to a browser.

**To Reproduce**

1. Create Markdown Visual note
2. Add link with URL to the note.
3. Try to visit the link from within the note.

**Expected behavior**

There should be some way to navigate to a URL from within a Markdown Visual Note.

**Screenshots**

When you click on link it switches to edit mode, but there is no way to navigate to it.

![click-link](https://user-images.githubusercontent.com/1482402/169667885-5b9f63dd-eddc-46c8-9d19-5b9187e166e1.png)

**Desktop (please complete the following information):**
 - Version 3.20.0

**Comment by dataprolet** [comment]
This is still an issue in 2024 on v3.192.13. C'mon guys, this is a simple feature, please fix this!
